web/src/app/admin/qualifications/qualifications.component.ts

```diff
@@ -42,7 +41,19 @@ export class QualificationsComponent implements OnInit {
       }
     }
   }
-
+  checkNameExits() {
```

Comment:
change function  name to checkQualificationExists

Reply:
done